test/build-turbopack-tests-manifest.js

```diff
@@ -0,0 +1,59 @@
+const fetch = require('node-fetch')
+const fs = require('fs')
+
+// TODO: Switch to nextjs-integration-test-data branch once https://github.com/vercel/turbo/pull/5999 is merged.
+const RESULT_URL =
+  'https://raw.githubusercontent.com/vercel/turbo/jrl-known-location-nextjs-integration-test-data/test-results/main/nextjs-test-results.json'
+const PASSING_JSON_PATH = `${__dirname}/turbopack-tests-manifest.json`
+const WORKING_PATH = '/home/runner/work/turbo/turbo/'
+
+async function updatePassingTests() {
+  const passing = { __proto__: null }
+  const res = await fetch(RESULT_URL)
+  const results = await res.json()
+
+  for (const result of results.result) {
+    for (const testResult of result.data.testResults) {
+      const filepath = stripWorkingPath(testResult.name)
+      for (const file of duplicateFileNames(filepath)) {
+        const fileResults = (passing[file] ??= {
+          passed: [],
+          failed: [],
+          pending: [],
+        })
+
+        for (const testCase of testResult.assertionResults) {
+          const { fullName, status } = testCase
+          const statusArray = fileResults[status]
+          if (!statusArray) {
+            throw new Error(`unexpected status "${status}"`)
+          }
+          statusArray.push(fullName)
+        }
+      }
+    }
+  }
+
+  fs.writeFileSync(PASSING_JSON_PATH, JSON.stringify(passing, null, 2))
```

Comment:
Technically it can't happen that test cases are removed from the list. But this might happen here due to 3 reasons:
- flaky tests
- tests being renamed or removed
- unmerged turbo Updates breaking test cases

I think we should handle them here.

Case 2 can be detected by a test case neither passing nor failing. In this case we can remove it.

Case 1 and 3 looks the same. Test case is failing but was passing before. I think we should keep it in the passing list and print a "potential flaky/broken test" to the console.

Reply:
done